dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigBootstrapTest.java

```diff
@@ -205,6 +206,9 @@ public void testConsoleSourceInEnvPropertiesXML() throws Exception {
             consolesourceList = new ArrayList<String>(Arrays.asList("trace"));
             checkConsoleLogUpdate(true, consoleLogFile, "INFO", consolesourceList, "");
 
+            //set server.xml to a basic config so that when server stops, it can successfully check for CWWKE0036I in console log
+            setServerConfig(SERVER_XML_BASIC);
```

Comment:
We have "CWWKE0036I" in JSON in this case, right?  This probably can be fixed by removing the colon at the end of "CWWKE0036I:" in 

https://github.com/OpenLiberty/open-liberty/blob/96e5f269e22dd0ee7bce7179cfb6cde6e0eeef43/dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java#L2388